sdk/cosmos/azure-cosmos/pom.xml

```diff
@@ -206,6 +206,16 @@ Licensed under the MIT License.
           <skip>true</skip>
         </configuration>
       </plugin>
+
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-checkstyle-plugin</artifactId>
+        <version>3.1.0</version> <!-- {x-version-update;org.apache.maven.plugins:maven-checkstyle-plugin;external_dependency} -->
+        <configuration>
+          <excludes>**/apachecommons/**/*,**/guava25/**/*,**/guava27/**/*,**/uuid/**/*</excludes>
```

Comment:
@alzimmermsft - how does the exclusion work ?
on groupId / artifiactId / version or some sort of regex ?
Because I see 2 guava exclusions, but only one apachecommons, however, there are 3 apache commons libraries used : https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/cosmos/cgmanifest.json

Reply:
This will exclude any files that are contained within a parent folder of `apachecommons`, `guava25`, `guava27`, or `uuid`. The three Apache commons libraries all fall under `apachecommons` but the Guava libraries have their own top level folders.

Reply:
Makes sense, thanks @alzimmermsft